Sources/Datadog/DatadogInternal/Codable/AnyEncodable.swift

```diff
@@ -0,0 +1,28 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+/// Type-erasure wrapper for any `Encodable` type to be transmitted through
+/// the message bus.
+public struct AnyEncodable: Encodable {
```

Comment:
Shouldn't this be yet `internal`? If not, then I believe "`AnyEncodable`" name might clash in some apps. How about calling it `DDAnyEncodable` in public API and having internal `typealias` to `AnyEncodable` for simplicity?

Reply:
Totally fine for having `public AnyEncodable` in `DatadogInternal`, but becase we don't yet have this module, we should be careful on naming public types. Using `DDAnyEncodable` + a `typealias` could work for now and be ultra cheap to replace in V2.